models/OrderModel.php

```diff
@@ -350,15 +352,20 @@ public function get_order_refunds( $order_id ) {
 
 		$response = array();
 
-		foreach ( $order_refunds as &$refund ) {
-			$values     = new \stdClass();
-			$values     = json_decode( $refund->meta_value );
-			$values->id = (int) $refund->id;
+		foreach ( $order_refunds as $refund ) {
+			$json_values = json_decode( $refund->meta_value );
```

Comment:
Variable name `$json_values` doesn't make any sense.
```suggestion
			$parsed_meta_value = json_decode( $refund->meta_value );
```